modules/administration/pages/monitoring.adoc

```diff
@@ -182,6 +179,9 @@ See https://github.com/wrouesnel/postgres_exporter.
 
 Installing and configuring exporters is done using a Salt formula.
 
+When you have the exporters installed and configured, you can enable and configure monitoring.
+You can do this directly through the {productname} {webui}, or set up service discovery.
```

Comment:
Here from my perspective it is not yet clear enough what exactly we mean with "you can enable and configure monitoring", I would guess it rather means "the monitoring server (Prometheus) can start scraping or gathering the metrics from monitored systems".

Further we can surely install and configure exporters directly through the web UI, but configuring Prometheus to scrape metrics from managed clients requires config changes on the monitoring server, so it would require at least a configuration channel assignment and a new revision of the config file for each monitored system that should be added. What I mean is: it does not really work as *directly* as it sounds here.

Reply:
Yeah, I think that's the general problem with this whole section, actually. We make it sound very straight forward, when it's in reality a fair bit more complicated. I updated the wording on that bit, anyway.